ncm-nscd/src/main/perl/nscd.pm

```diff
@@ -91,6 +109,18 @@ sub Configure {
 
     # check per-service options we know about
     my @services = ('passwd', 'group', 'hosts');
+
+    my $nscd_version = $self->_get_nscd_version();
+
+    if ($nscd_version >= 2.12) {
```

Comment:
Perl is magic 🤩

The method returns a string. >= is a numeric operator. I can believe Perl is clever enough to cast $nscd_version automatically, and this will work nscd gets to version 2.100 which is < 2.12. Seems safe enough, I think!